spec/features/navbar/navbar_spec.rb

```diff
@@ -1,14 +1,20 @@
 require 'rails_helper'
 
 RSpec.describe 'Navbar' do
-  describe 'visit /' do
-    it "can see the navbar and its links" do
-      visit "/"
+  describe 'visit root path' do
+    before :each do
+      visit root_path
 
+      @landscape1 = Photo.create!(title: "Beautiful Mountains", description: "Look at dem mountains", genre: "landscape", url: 'https://cdn.pixabay.com/photo/2016/05/24/16/48/mountains-1412683__340.png')
```

Comment:
It's so odd that we had to put these objects into the navbar test. I will have to follow up after doing some research!